cluster/common.sh

```diff
@@ -277,14 +285,15 @@ function load-or-gen-kube-basicauth() {
 #   KUBE_VERSION
 function set_binary_version() {
   if [[ "${1}" =~ "/" ]]; then
-    IFS='/' read -a path <<< "${1}"
+    IFS='/' read -a -r path <<< "${1}"
     if [[ "${path[0]}" == "release" ]]; then
       KUBE_VERSION=$(gsutil cat "gs://kubernetes-release/${1}.txt")
     else
       KUBE_VERSION=$(gsutil cat "gs://kubernetes-release-dev/${1}.txt")
     fi
   else
     KUBE_VERSION=${1}
+    export KUBE_VERSION
```

Comment:
Can you change the comment of this function to say that it exports this var? eg:
```
# Args:
#   $1 version string from command line
# Vars exported:
#   KUBE_VERSION
function set_binary_version() {
```

Reply:
thanks for your comments, I have modified those.